src/web/mjs/connectors/Kuman5.mjs

```diff
@@ -29,7 +22,7 @@ export default class Kuman5 extends Connector {
 
     async _getMangasFromPage(page) {
```

Comment:
In case of the placeholder solution this method no longer needs to be overwritten, the base from template should work